x-pack/plugins/security_solution/public/detections/pages/alerts/index.tsx

```diff
@@ -24,8 +26,15 @@ const AlertsRoute = () => (
 );
 
 const AlertsContainerComponent: React.FC = () => {
+  const { executionContext } = useKibana().services;
   useReadonlyHeader(i18n.READ_ONLY_BADGE_TOOLTIP);
 
+  // Application ID and current URL are traced automatically.
+  useExecutionContext(executionContext, {
+    page: SecurityPageName.alerts,
+    id: 'all',
```

Comment:
You don't need to add the `id` if there's not a specific entity that this points to